barretenberg/cpp/src/barretenberg/ultra_honk/oink_prover.cpp

```diff
@@ -149,6 +153,14 @@ template <IsUltraFlavor Flavor> void OinkProver<Flavor>::execute_grand_product_c
     transcript->send_to_verifier(domain_separator + commitment_labels.z_lookup, witness_commitments.z_lookup);
 }
 
+template <IsUltraFlavor Flavor> typename Flavor::RelationSeparator OinkProver<Flavor>::generate_alphas()
+{
+    RelationSeparator alphas;
+    for (size_t idx = 0; idx < alphas.size(); idx++) {
+        alphas[idx] = transcript->template get_challenge<FF>("Sumcheck:alpha_" + std::to_string(idx));
```

Comment:
this is not necessarily for sumcheck (and neither are the gate challenges) so maybe it should be made explicit?